client/javascripts/components/discussions/discussions.tag

```diff
@@ -0,0 +1,142 @@
+<discussions>
+<div class="discussion" each={ discussion in discussion_page.data } id="discussion-{ discussion.id}">
+  <div class="discussion__title">
+    <h5>{ discussion.title }</h5>
```

Comment:
Heading numbers should be are relative to semantic elements like ```header```, ```footer```, ```main```, ```section``` or a custom element like ```discussions```, so in this case it'd be a ```h1``` or ```h2```.